### ISSUE TITLE
feat: protect users against ReDoS

### ISSUE THREAD

**Issue by NanderTGA** [NanderTGA/msgroom-orm#145]
ReDoS vulnerabilities are a serious problem and people using regex prefixes could fall victim to this.
I want to protect people against this by checking for ReDoS vulnerabilities in the background using [recheck](https://makenowjust-labs.github.io/recheck/).

It does add an extra dependency though.
Do or don't?

**Comment by nolanwhy** [comment]
yes, but if you can, do something so we can disable it in the Client, like new Client("testbot", [ "e" ], { recheck: false })

**Comment by NanderTGA** [comment]
> yes, but if you can, do something so we can disable it in the Client, like new Client("testbot", [ "e" ], { recheck: false })

Yeah, I was planning to do that.
